packages/react-app/src/views/QuadraticDiplomacyVote.jsx

```diff
@@ -1,176 +1,234 @@
-import { Button, Checkbox, Divider, Space } from "antd";
-import React, {useEffect, useState} from "react"
+import React, { useEffect, useState, useMemo, useCallback } from "react";
+import { Button, Checkbox, Divider, Space, List, Steps, Typography, Badge } from "antd";
+import { SmileTwoTone, LikeTwoTone, CheckCircleTwoTone, MinusOutlined, PlusOutlined } from "@ant-design/icons";
+import { Address } from "../components";
+const { Title } = Typography;
 
-export default function QuadraticDiplomacyVote({
-  voteCredits,
-  contributorEntries,
-  tx,
-  writeContracts,
-}) {
-  const [contributors, setContributors] = useState({});
+export default function QuadraticDiplomacyVote({ voteCredits, contributorEntries, tx, writeContracts }) {
+  // local state
   const [selectedContributors, setSelectedContributors] = useState({});
-  const [currentPage, setCurrentPage] = useState(1);
+  const [currentStep, setCurrentStep] = useState(0);
```

Comment:
Like the name change to "Step" ;)

I know we are used to always start at 0, but in cases like this, I prefer to start at 1:
- Since we are already in the first step when we load this Voting screen, 0 is a falsey value and could cause some futures issues.
- If we were to display in the UI in which step the user is, we would have to do `currentStep + 1` (a bit confusing)

Anyway this is just a personal preference :P Let me know what you think.

Reply:
I wanted to start at 1 as well! But actually the ant-design's [`Steps`](https://ant.design/components/steps/) component needs 0 to start with. I did try setting the `<Steps initial={1} />` but for some reason it didn't work as expected. Will read their docs again and see if this is possible.